### ISSUE TITLE
SVGPathPen rendering TTF glyphs vertically mirrored

### ISSUE THREAD

**Issue by wolfgang42** [fonttools/fonttools#2388]
I wrote a simple program to extract glyphs:
```python3
from fontTools.ttLib import TTFont

ttf = TTFont('/usr/share/fonts/truetype/lato/Lato-Regular.ttf')
gs = ttf.getGlyphSet()

from fontTools.pens import svgPathPen
p = svgPathPen.SVGPathPen(gs)
gs['R'].draw(p)
print(p.getCommands())
```

But when I put the resulting path in an SVG it comes out upside-down!
```svg
<svg version="1.1" baseProfile="full" width="1253" height="1500" xmlns="http://www.w3.org/2000/svg">
  <path fill="black" d="M376 598V0H183V1433H588Q724 1433 823.0 1405.5Q922 1378 986.5 1326.0Q1051 1274 1082.0 1200.5Q1113 1127 1113 1036Q1113 960 1089.0 894.0Q1065 828 1019.5 775.5Q974 723 908.5 685.5Q843 648 760 630Q798 606 824 569L1242 0H1070Q1017 0 992 41L620 553Q603 577 583.5 587.5Q564 598 523 598ZM376 739H580Q664 739 728.5 759.5Q793 780 836.5 817.5Q880 855 902.0 907.0Q924 959 924 1022Q924 1150 840.0 1215.0Q756 1280 588 1280H376Z" />
</svg>
```
![R](https://user-images.githubusercontent.com/2261204/128282144-7a029a98-93f7-4d47-87c2-39de66e25aa5.png)

Is this intentional? I'm guessing it's caused by SVG's coordinate system (where Y increases going down) vs PostScript's (where Y increases going up). Correcting it post-facto is straightforward (just add `transform="scale(1,-1)" transform-origin="center"` to the `<path>`) but this is a pretty confusing result.

**Comment by ACCia** [comment]
You can try this
```
p = svgPathPen.SVGPathPen(gs)
tpen= TransformPen(p  , (1, 0, 0, 1, 0, 0))
gs['R'].draw(tpen)
print(p.getCommands())
```
If the direction is wrong.Please modify TransformPen parameter.

**Comment by wolfgang42** [comment]
Yes, that's mainly what I ended up doing. Actually, I also had to correct for font scaling, so the workaround I came up with was approximately:
```python
unitsPerEm = ttf['head'].unitsPerEm
ttf2em = transform.Identity.scale(1/unitsPerEm, 1/unitsPerEm)
svgPerEm=100
xform = transform.Identity.translate(0, svgPerEm).scale(svgPerEm, -svgPerEm).transform(ttf2em)
p = svgPathPen.SVGPathPen(ttf.getGlyphSet()['g'])
tp = transformPen.TransformPen(p, xform)
glyph.draw(tp)
print(p.getCommands())
```

But perhaps this is mainly a documentation problem. Is the intent that each type of pen and outline uses its “native” coordinate system? If so:
* It would be very helpful to document on each pen and outline which coordinate system it uses.
* Some helper functions to translate between each coordinate system and a standardized intermediate system (e.g em-based and with a defined orientation), in much the same way as a Pen is an intermediate between outlines and drawings, would be quite useful.
